.vortex/cli/src/Utils/Tui.php

```
@@ -124,20 +135,77 @@ public static function getChar(): string {
 
-    // Restore terminal settings.
-    system('stty -cbreak echo');
+    foreach ([SIGINT, SIGTERM] as $signal) {
+      // @codeCoverageIgnoreStart
+      pcntl_signal($signal, static function (int $received) use ($restore): void {
+        $restore();
 
-    return $char;
+        // The conventional status for a run a signal ended, so a caller reading
+        // the exit code still learns which signal it was.
+        exit(128 + $received);
```

PHPMD (2.15.0)

[error] The method trapSignals() contains an exit expression. (undefined)

(ExitExpression)